### ISSUE TITLE
[Markup and meta data] HTML markup isn't valid. (09.01.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#33881]
#### VFS team name
AFS Education

#### VFS product name
GI Bill Comparison Tool

#### Point of Contact/Reviewers
Noah Gelman (@noahgelman) - Accessibility

---

### Platform Issue
HTML markup isn't valid.

### Issue Details
The HTML of card markup uses incorrect HTML. The `id` of the classification header begins with a number. Ids cannot start with numbers. The `a` tag does not need the `aria-labelledby` property to point to it's own content. The `h3` inside the `a` tag does not need the `aria-label`. The `aria-label` value is the same as it's own value. All of this was done to add the classification to the end of the card link. Just add the classification to the end of the link as screenreader only text. Same thing for the Compare checkbox. Remove all the aria-labels. Put whatever labelling that is necessary inside the label element. If it needs to be screen reader only then put it in screenreader only text. The `name` and `id` attributes on the Compare checkbox and label are not unique. Some card titles are the same, but in different locations. Doing a search for `UNIVERSITY OF HEALTH SCIENCES AND PHARMACY IN ST LOUIS` has 2 `UNIVERSITY OF HEALTH SCIENCES AND PHARMACY IN ST LOUIS` results and therefore their `id` and `name` attributes are the same.

### Link, screenshot or steps to recreate
Search `UNIVERSITY OF HEALTH SCIENCES AND PHARMACY IN ST LOUIS` and look at results

### VA.gov Experience Standard
[Category Number 09, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.1_A

---

### Platform Recommendation
Remove the uneeded aria attributes and move screenreader text to an element with the `sr-only`. Change the ids beginning with numbers. Make the Compare checkbox id and name attributes unique

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by AdamFreemer** [comment]
@shiragoodman Hi Shira! Question on one particular part of this one:

"The a tag does not need the aria-labelledby property to point to it's own content."

Here's an example section for discussion on:
```
<a aria-labelledby="10511717-label 10511717-classification" href="/gi-bill-comparison-tool-sandbox/institution/10511717">SPRINGFIELD POLICE DEPARTMENT</a>
```

What are we looking for in the `aria-labelledby` content exactly? The rest of this task seems clear, just this particular one I was looking for a little more clarity on.

**Comment by shiragoodman** [comment]
@noahgelman are you able to answer Adam's question above?

**Comment by noahgelman** [comment]
@AdamFreemer The `aria-labelledby` property is pointing to 2 things.
1. It's own parent, the `h3`. It doesn't need to do this. It's pointing to itself.
2. It's pointing to the classification. It doesn't need to do this. Remove it. Add the classification to the link as screen reader only text

**Comment by AdamFreemer** [comment]
@noahgelman so remove the `aria-labelledby` property from the tag completely? Just being sure that's what you're saying? And for clarity (in this example) `10511717-classification` as a screen reader only property?

**Comment by noahgelman** [comment]
Yes, remove it entirely. The content of the classification element can be
added to the `a` as screen reader only text

**Comment by AdamFreemer** [comment]
One more question @noahgelman @shiragoodman 

"The name and id attributes on the Compare checkbox and label are not unique. Some card titles are the same, but in different locations."
- Would facility code work here for uniqueness? It's used elsewhere.  
```
id={`${facilityCode}-classification`}
```

**Comment by noahgelman** [comment]
as long as there are no duplicates and it doesn't start with a number

**Comment by jfinnern** [comment]
Validated in staging
Test run: https://dsvavsp.testrail.io/index.php?/runs/view/2742&group_by=cases:title&group_order=asc
Test case:
https://afs365.sharepoint.com/:x:/r/sites/DGIBillServicesProject/Shared%20Documents/Legacy%20AO/Team%203/QA/Test%20Results/AFS-Sprint%204/%5BMarkup%20and%20meta%20data%5D%20HTML%20markup%20isn%27t%20valid.%20(09.01.1).xlsx?d=w2a53bc456a51436c832e098bc0433338&csf=1&web=1&e=MxLRKc

**Comment by ERRonat0609** [comment]
@mattpointzxer0 Please review and approve. You can close this ticket once it has been validated.

**Comment by mattpointzxer0** [comment]
🏁

**Comment by mrblanco1** [comment]
Hi @noahgelman and @shiragoodman - After having reviewed this ticker with EDU they have a question I think you can help:
"ticket #33881 – adding classification to differentiate schools with identical names - 
Question: how does the user know what the additional classification means? It doesn’t seem like adding this helps the user to understand what the differences are between two schools with the same name."

When you get a chance, can you please provide feedback so we can pass along? Joe Preisser and his team would like to better understand please.

Thanks!
